spec/features/obituaries/create_spec.rb

```diff
@@ -89,6 +89,24 @@
 
     expect(page).to have_content("First name can't be blank and Last name can't be blank")
   end
+
+  it "can enter an obituary over 200 characters, but it will only display full length on show page", :vcr do 
```

Comment:
I limited the description to 200 characters, if we want to expand that, it would be a simple fix.

Reply:
These are randomly failing(I believe that is what is causing the build to fail), whenever the page renders the NYT results before the community tab in the search index, @jpc20, do you know how to test only the community tab?

Reply:
Hmm... I can check this out. It should always be rendering the community results by default, but I can try to figure out what's going on.